composer/autoload.php

```diff
@@ -1,6 +1,8 @@
 <?php
-
-// autoload.php @generated by Composer
+/**
```

Comment:
Can we rather specify this through the .reuse/dep5 file? Those files are autogenerated by composer to have a static autoloader and the CI checks if those match the generated ones.

Reply:
Yes, sure, especially when auto generated 👍

Reply:
@juliushaertl DONE 👍